### DISCUSSION TITLE
Merge branch "develop" into "master"

### DISCUSSION THREAD

**Discussion by electron271** [logisim-evolution/logisim-evolution#1495] [Dev talk]
I think that the `develop` branch should be merged into `master`, or be set as the default branch, as since the last commit on the `master` branch was a year ago, so seeing that, users may think that Logisim Evolution is not actively developed, and old.

**Comment by R3dst0ne** [comment]
`master` is always the current release of Logisim-evolution. `develop` gets merged into `master` once it is ready to be released as a new version (in this case: `3.8`).
There might be better ways of doing this, but this is the way it currently is.

**Comment by electron271** [comment]
I still think that develop should at least be set as the master branch, as people *(including me at first)* may think that the project isn't in active development, or know how to contribute. Another issue is that people wanting to build it from scratch, will most likely accidentally build the wrong version, instead of the development version, as people doing that usually want.
